README.md

```
@@ -300,6 +300,49 @@ Settings are managed in `~/.claude-mem/settings.json` (auto-created with default
+```json
+{
+  "CLAUDE_MEM_MODE": "code--zh"
+}
+```
+
+Modes are defined in:
+
+```text
+plugin/modes/
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)